src/providers/bttv/BttvEmotes.cpp

```diff
@@ -179,8 +179,10 @@ void BttvEmotes::loadChannel(std::weak_ptr<Channel> channel,
         shared->addMessage(makeSystemMessage(sysMessage));
     }
 
+    const int timeout = 2 ^ retryCount * 1000;
```

Comment:
Isn't this doing XOR on `2` and `retryCount * 1000`?

Reply:
Dumb mistake from a non-cpp dev. Resolved